txpool/txpool_test.go

```diff
@@ -3709,14 +3711,55 @@ func TestBatchTx_SingleAccount(t *testing.T) {
 	// mutex for txHashMap
 	mux := &sync.RWMutex{}
 	counter := uint64(0)
-
+	enqueuedCount := 0
+	promotedCount := 0
+	ev := (*proto.TxPoolEvent)(nil)
 	wg := sync.WaitGroup{}
-	wg.Add(int(defaultMaxAccountEnqueued))
+	wg.Add(1)
```

Comment:
Optional nitpick (I find it slightly more elegant in terms of declaring `ev *proto.TxPoolEvent`, that's why I'm suggesting such change)
```suggestion
	var (
		mux           sync.RWMutex
		counter       uint64
		enqueuedCount int
		promotedCount int
		ev            *proto.TxPoolEvent
		wg            sync.WaitGroup
	)
```